src/esp/gfx/DrawableConfiguration.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Meta Platforms, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_GFX_DRAWABLECONFIGURATION_H_
+#define ESP_GFX_DRAWABLECONFIGURATION_H_
+
+#include <Magnum/Resource.h>
+#include "esp/gfx/PbrIBLHelper.h"
+#include "esp/gfx/SkinData.h"
+#include "esp/metadata/attributes/PbrShaderAttributes.h"
+namespace esp {
+namespace gfx {
+class DrawableGroup;
+
+/**
+ * This class will hold configuration values and utilities for Drawables and the
+ * shaders that they own.
+ */
+class DrawableConfiguration {
```

Comment:
This was made a class and not a struct because I foresee functionality being added/moved from RM in the future.